### DISCUSSION TITLE
求出东方系列游戏progect

### DISCUSSION THREAD

**Discussion by mmmgpx** [Gamuxorg/bbs#430] [投递和请求游戏]
想要打东方绯想天和东方非想天则 和东方永夜抄，不方便移植的话，可以直接用 红酒
渣机能玩就行
cpu: amd a8 5500
GPU:gpx650
4g内存
https://cloud.lilywhite.cc/s/4ZUW?path=%2F
此链接是东方狗网站
可以下到WINDOWS版
跪求移植或者wine

**Comment by ChocoFleece** [comment]
国内非想天则社区Linux支持请见：
https://wiki.514.live/FAQ/Failed-to-Start/Linux.html
